custom/abt/reports/flagspecs.yaml

```diff
@@ -779,14 +779,14 @@ http://openrosa.org/formdesigner/7B543EA4-A416-4E46-A121-C4F57412B5AA:
   - question: [q29_action]
     flag_name: "Pesticide usage inventory?"
     flag_name_fr: "Pesticide inventaire d'utilisation?"
-    warning: "Problem reported: Pesticide inventory does not balance."
+    warning: "Problem reported: Pesticide packaging waste inventory does not balance."
     warning_fr: "Problème signalé : L’inventaire du ou des pesticides ne s’équilibre pas."
   - question: [q30_action]
     flag_name: "Old waste stored?"
     flag_name_fr: "Old déchets stockés ?"
-    warning: "Problem reported: Surplus pesticide in inventory."
+    warning: "Problem reported: More than one season of accumulated solid waste in store."
     warning_fr: "Problème signalé : Pesticides excédentaires dans l’inventaire."
-  - question: [q31_action]
+  - question: [q31_action]m
```

Comment:
I don't think that would be legal yaml (or maybe it would cause that to be interpreted as a string instead of a list)